### PR TITLE
Apple page

### PR DESCRIPTION
**Pull request by dannkkie**
Solo project done please review

### CODE REVIEW

**Review by Forison — COMMENTED** [review]
# Pull request under review by Addo

**Comment by dannkkie** [comment]
Okay

**Review by Forison — CHANGES_REQUESTED** [review]
# Hi @Edmund👏 
You have done a good job so far.🙂
This is the output I get from your project🙂
Screenshot from 2019-11-05 18-31-09
Screenshot from 2019-11-05 18-31-14

#### Here are a few changes we have to make to get this project approved🙂
- [x] It should be made of nine different buttons with the Apple logo on the leftmost section and a search field on the rightmost section🙂
Let us replace this ```<a href="#"><div class="store"></div></a>``` with ```<a href="#">store</a>``` and so on
- [x] The main content should have an image set as background (as CSS [`background-image`]
currently it is blank🙂

## That is it Edmund, Kindly do these changes and submit for view🙂
feel free to contact me on [LinkedIn](https://www.linkedin.com/in/forison-addo-558201159/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. I would be happy to assist you.
#### Happy coding🙂

**Comment by dannkkie** [comment]
okay doing that now

**Comment by dannkkie** [comment]
For the hero image you are not seeing i doubt if it is from me because everything is showing well here

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+#copyright li {
+  list-style-type: none;
+  margin-right: 3px;
+  margin-left: 3px;
+  padding-left: 3px;
+  padding-right: 5px;
+}
+
+#nav-links {
+  background-image: url("/images/nav-surround.png");
```
Comment (Redvanisation) [inline css/styles.css]: The hero image is not showing, try this instead `url("../images/nav-surround.png")`

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  🛑

### Hello @dannkkie   🌟

**Good work implementing the changes, but we need to fix the issues below before moving on ⬇️** 

1. The nav buttons don't have text, please have the nav looking like the original page.

2. The hero background image is not showing, check my comment below on how to fix that.

3. the 4 bottom elements under the image should be links.

4. The footer link aren't aligned, please align them like on the original page.

### _After making the required changes please submit a new code review request_

### Good luck guys! 🍀

**Comment by dannkkie** [comment]
Please check and review kindly check your browser because the footer is well arranged just like in the original page and the 4 images are links you can inspect the code. All other changes have been made accordingly

**Comment by ebukaume** [comment]
## Changes Required! :stop_sign: 

Hi @dannkkie :wave: 

I recommend you consider @Redvanisation suggestion.
> The hero image is not showing, try this instead url("../images/nav-surround.png")

I see this when I view your page using raw githack as well as when using my local file system.
review-apple-dan

However, when I spin up live-server, I get this :point_down:  (not taking full width)
review-apple-dan2

Please refer to @Redvanisation's last review and make the requested fixes. Your images (4) are not behaving like links _(clickable, pointer cursor, etc)_. Kindly fix those too.

**Submit another code review request after making those changes**


Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Comment by dannkkie** [comment]
The live page we are supposed to clone doesn't take full width also i have made changes to this url("../images/nav-surround.png") as directed by @Redvanisation

**Comment by shshamim63** [comment]
### :arrow_forward:  Status Approved!! :arrow_backward: 
___
Hi @dannkkie :wave: ,
All the part that are required to declare this project are in here. Nice job. :clap: 
#### This :+1:  is from me, you can merge this and don't forget to fill the form.
___
### Happy Coding :herb: 
#### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

**Comment by dannkkie** [comment]
Thank you @shshamim63

**Pull request merged**